vllm/attention/selector.py

```diff
@@ -57,6 +57,8 @@ def get_attn_backend(
             ROCmFlashAttentionBackend)
         return ROCmFlashAttentionBackend
     elif backend == _Backend.TORCH_SDPA:
+        assert is_cpu(), RuntimeError(
+            "Torch SDPA backend is only used for the CPU device.")
```

Comment:
QQ: Won't it be also used by the XPU backend?

Reply:
Right, but the XPU backend is not merged for now, maybe I can leave a note here.